tools/Config.mk

```diff
@@ -467,12 +486,13 @@ define CLEAN
 	$(Q) if exist *$(LIBEXT) (del /f /q *$(LIBEXT))
 	$(Q) if exist *~ (del /f /q *~)
 	$(Q) if exist (del /f /q  .*.swp)
-	$(Q) if exist $(OBJS) (del /f /q $(OBJS))
+	$(Q) $(foreach OBJ, $(OBJS), $(if exist $(OBJS) (del /f /q $(OBJ))))
 	$(Q) if exist $(BIN) (del /f /q  $(BIN))
 endef
 else
 define CLEAN
-	$(Q) rm -f *$(OBJEXT) *$(LIBEXT) *~ .*.swp $(OBJS) $(BIN)
+	$(Q) rm -f *$(OBJEXT) *$(LIBEXT) *~ .*.swp $(BIN)
+	$(Q) $(foreach OBJ, $(OBJS), $(rm -f $(OBJ)))
```

Comment:
can't use MAX_ARGS here as well?

Reply:
We could for the non windows case, but for the windows case it is not a list of files at the end as arguments we need it twice.
The performance difference was negligible here so I went this was for consistency.